docs/advanced/hdd-support.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 2
+sidebar_label: HDD Support
+title: ""
+keywords:
+  - HDD
+  - StorageClasses
+  - StorageTiering
+  - Tiering
+  - ColdStorage
+---
+
+# HDD Support
+
+In harvester v1.0, *StorageClass* is introduced. That also introduces the disk tag and node tag for it. With the above things, **HDD** is also supported with this version.
```

Comment:
```suggestion
Harvester v1.0 introduces *StorageClass* as well as the disk tags and node tags for it. With these features, Harvester v1.1 now supports **HDD**.
```

Reply:
Does it mean harvester v1.1 now supports HDD?

Reply:
yes!  done for the above suggestions